src/amo/components/AddonReviewList/index.js

```diff
@@ -289,6 +317,16 @@ export class AddonReviewListBase extends React.Component<InternalProps> {
       );
     }
 
+    let featuredReviewHeader;
+    if (featuredReview) {
+      featuredReviewHeader =
+        siteUser && featuredReview.userId === siteUser.id
+          ? i18n.gettext('My Review')
```

Comment:
I don't think it is necessary to customize the title for owners. It can always say `Review by %(userName)s`.

Reply:
Sure, that sounds reasonable.